nuxt.config.ts

```diff
@@ -123,6 +125,8 @@ export default defineNuxtConfig({
     public: {
       serverUrl: SERVER_BASE_URL,
       mercadoPagoPublicKey: process.env.MERCADO_PAGO_PUBLIC_KEY,
+      srcSiteKey: process.env.SRC_SITE_KEY,
```

Comment:
```suggestion
      srcSiteKey: process.env.RECAPTCHA_SECRET_KEY,
```

Você acha que assim ficaria mais explicito?

Reply:
tinha deixado do jeito que o site do google sugere, mas já atualizei nesse commit - 22dc09480bb0ef125e6de1a7c54f64466475b8ef